cmd/osv-scanner/__snapshots__/main_test.snap

```diff
@@ -141,6 +141,43 @@ Loaded filter from: <rootdir>/fixtures/go-project/osv-scanner.toml
 
 ---
 
+[TestRun/PURL_SBOM_case_sensitivity_(api) - 1]
```

Comment:
the fact that the offline output has vulnerabilities whereas the online one doesn't indicates there's a potential bug in the api's Alpine comparator 🤔

Reply:
Yes there is a bug in alpine version enumeration, just fixed this week in: https://github.com/google/osv.dev/pull/2241 but not in production yet.

Reply:
Cool - I'd seen a few of those go around this/last week, so figured it might have been known 😅